SFconv/SFconv.cpp

```diff
@@ -101,15 +101,15 @@ Utf8ToString(const char* s)
 	utf16::iterator out = buf;
 	utf8::const_iterator in  = s, 
 	                     end = s + len;
-    while (in != end && !(in.error() || out.error()))
+	while (in != end && !(in.error() || out.error()))
         *out++ = *in++;
     
 	if (in.error() || out.error()) {
```

Comment:
The cause of the warning you're seeing is indentation of the block guarded by this line, it's indentation what the compiler's complaining about as it does indeed make it appear to part of the while loop's body when it's not.